lib/br_danfe/helper.rb

```diff
@@ -43,5 +43,16 @@ def self.nfe?(xml)
     def self.format_cep(cep)
       cep.sub(/(\d{2})(\d{3})(\d{3})/, '\\1.\\2-\\3')
     end
+
+    def self.format_datetime_with_timezone(xml_datetime)
```

Comment:
E se a gente não tivesse e método e sim criar um parâmetro no outro método chamado `format_datetime` chamado `with_time_zone: (false - padrão | true)`?

Inclusive o código daquele cara poderia ser assim:

```Ruby
xml_datetime.present? ? Time.parse(xml_datetime).strftime(format) : ''
```

Dai o `format` a gente vai ter o padrão e se for `with_time_zone`, adiciona o `%:z` no final. O que acham?